arrays/task1.js

```diff
@@ -0,0 +1,23 @@
+// Count of positive, negative and zero numbers in the array
+
+const numbers = [2, -5, 0, 7, -3, 0, 10, -8];
+let positiveCount = 0;
+let negativeCount = 0;
+let zeroCount = 0;
+for (let i = 0; i < numbers.length; i++) {
+  if (numbers[i] > 0) {
+    positiveCount++;
+  }
+  if (numbers[i] < 0) {
+    negativeCount++;
+  }
+  if (numbers[i] === 0) {
+    zeroCount++;
+  }
```

Comment:
in this case you either need to add `continue`  in each `if` block or use `else if`

why? because even if number is positive you still do perform other checks but we know they never be truthy

Reply:
added 'else if' instead of 3 if statements